### ISSUE TITLE
Parse eror with 4.6.0-rc1 and master

### ISSUE THREAD

**Issue by ibennetch** [phpmyadmin/phpmyadmin#12055]
I guess we'll release an -rc2 tomorrow :)

> PHP Parse error:  syntax error, unexpected 'echo' (T_ECHO) in /var/www/pma-dev/templates/database/structure/sortable_header.phtml on line 69,

I guess it's related to this duplication:
```
68 echo PMA_Util::linkOrButton(
69 echo PMA\libraries\Util::linkOrButton(
```

**Comment by zixtor** [comment]
Seems like a merge issue in https://github.com/phpmyadmin/phpmyadmin/commit/a29f154f37400264e8b7fbbbf4cf3bf1d046f127

**Comment by ibennetch** [comment]
I've just released version 4.6.0-rc2 to address this. Thanks.
